supervisor/backups/manager.py

```diff
@@ -227,15 +230,10 @@ async def _do_restore(
         self,
         backup: Backup,
         addon_list: list[str],
-        folder_list: list[Path],
+        folder_list: list[str],
         homeassistant: bool,
         replace: bool,
     ):
-        # Version 1
-        if FOLDER_HOMEASSISTANT in folder_list:
-            folder_list.remove(FOLDER_HOMEASSISTANT)
-            homeassistant = backup.homeassistant_version is not None
-
```

Comment:
`do_restore_full` doesn't seem to have code which handles `FOLDER_HOMEASSISTANT` like this. You added it to `do_restore_partial`, should `do_restore_full` have it as well? Now that `_do_restore` isn't handling it for backups anymore.

Reply:
Oh I get it. It's a full backup so we just set the `homeassistant` input in `_do_restore` to true, we don't need to check. Right.